config/config.go

```diff
@@ -2,23 +2,25 @@
 package config
 
 import (
-	"sync"
-
+	"errors"
+	"github.com/companieshouse/chs.go/log"
 	"github.com/companieshouse/gofigure"
+	"sync"
 )
 
 var cfg *Config
 var mtx sync.Mutex
+var mandatoryElementMissing bool
 
 // Config defines the configuration options for this service.
 type Config struct {
-	BindAddr          string   `env:"BIND_ADDR" flag:"bind-addr" flagDesc:"Bind address"`
-	BrokerAddr        []string `env:"KAFKA_BROKER_ADDR" flag:"broker-addr" flagDesc:"Kafka broker address"`
-	SchemaRegistryURL string   `env:"SCHEMA_REGISTRY_URL" flag:"schema-registry-url" flagDesc:"URL for Schema Registry"`
-	OfficerDeltaTopic string `env:"OFFICER_DELTA_TOPIC" flag:"officer-delta-topic" flagDesc:"Topic for the officer delta"`
+	BindAddr          string    `env:"BIND_ADDR" flag:"bind-addr" flagDesc:"Bind address"`
+	BrokerAddr        []string  `env:"KAFKA_BROKER_ADDR" flag:"broker-addr" flagDesc:"Kafka broker address"`
```

Comment:
This is an array. Could there be more than one address? If so, what form should the config take?

Reply:
For the above, we would expect it to be read in as a comma separated list of strings

Reply:
Cool. So perhaps: "Kafka broker addresses (comma separated if more than one)"?